html-aam/roles-minimum.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang=en>
+<head>
+  <title>HTML-AAM Minimum Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+  <p>Tests minimum computedrole mappings defined in <a href="https://w3c.github.io/html-aam/#minimum-role">HTML-AAM</a>, where the returned computed role for generic elements is expected to change based on the generic elements having attributes that would require the element no longer be generic, and potentially ignored by browsers. Most test names correspond to a unique ID defined in the spec.<p>
```

Comment:
Since this isn't landed yet (dead spec link), I'd expect to see a link to the issue that cross-references the proposed spec diff, implementation trackers, etc.